.concourse/bookinfo-watch.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# This script runs the commands passed to it in parallel to a curl against the bookinfo application
```

Comment:
If it doesn't have anything concourse-specific ( and it doesn't seem to have ), please move under test directory or bin.